core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -1420,6 +1420,15 @@ class KafkaZkClient private[zk] (zooKeeperClient: ZooKeeperClient, isSecure: Boo
     zooKeeperClient.close()
   }
 
+  /**
+   * Wait up to the provided timeoutMs this KafkaZKClient to close.
+   * Supplying a timeoutMs of 0 means to wait forever.
+   */
+  def closeAndWait(timeoutMs: Int): Boolean = {
+    zooKeeperClient.closeAndWait(timeoutMs)
+  }
+
+
```

Comment:
Did you mean to have 2 empty lines?

Reply:
No, that should have been 1 line. Thanks!